examples/sshguard.conf.sample

```diff
@@ -36,6 +35,9 @@ IPV6_SUBNET=128
 # Size of IPv4 subnet to block. Defaults to a single address, CIDR notation. (optional, default to 32)
 IPV4_SUBNET=32
 
+# Subsequent blocks increase by this factor. (optional, default 2)
```

Comment:
I would move this right after `BLOCK_TIME` so that related options are together.

Reply:
Perhaps we can come up with a better name for this variable. If the initial block time is 1 minute, it is unclear if setting `INCREASING_FACTOR` to 2 results in the next block being 2 minutes (1×2 = 2) or 3 minutes (1 + 1×2 = 3, an *increase* of 2×).